### DISCUSSION TITLE
Отзыв о работе 1.5....

### DISCUSSION THREAD

**Discussion by SkipTyler282** [Flowseal/zapret-discord-youtube#351] [General]
Привет, флоусил. Ты просил в описании к 1.5.0 рассказать как работает. Ну, понятно 1.5.1 запускал.

Так вот. Дискорд жив везде. Ютуб по разному, а именно:
основной general - ютуб мёртв, первая альтернатива - мёртв, вторая альтернатива - еле еле, третья и четвёрвая альтернативы - ютуб хорошо себя чувствует (и в 3ей даже дискорд несколько быстрее), пятая - ютуб мёртв. Оператор связи - Связьинформ (некоторые регионы центральной России). Надеюсь, эта инфа будет хоть как-то полезна.

И, если приделаешь, чтобы работал инстаграм и протонмейл (особенно он), вообще буду готов задонатить!

**Comment by Flowseal** [comment]
я там просил дать фидбек если что-то не работал по сервис инсталлеру, но спасибо

**Comment by demon19912708** [comment]
25.10.2024 полностью не работает Крым
